dev/django.Dockerfile

```diff
@@ -18,3 +18,6 @@ RUN pip install --editable /opt/django-project[dev]
 
 # Use a directory name which will never be an import name, as isort considers this as first-party.
 WORKDIR /opt/django-project
+
+COPY atlascope_plugins /opt/atlascope-plugins/
+RUN pip install --editable /opt/atlascope-plugins/*
```

Comment:
How does pip watch for updates? And does that mean we *won't* need to do a Docker rebuild upon changing importer code?

Reply:
Yknow I'm glad you asked, because until now I was trusting something in the docs for `--editable` that I apparently misinterpreted. I _thought_ that this flag would take care of updates, but I suppose it doesn't work as well with docker as I thought. I only tried it just now, and it didn't do what I expected. 

It does take a build to get changes to the plugin. Perhaps we can figure out that special treatment in another PR? I'll file an issue.

Reply:
I think `--editable` generally makes your installation immediately responsive to changes in the source files (since it is using symlinks or similar to "install" your package). But since we need to actually copy the files into the docker image before they can be pip-installed (or, something along these lines), we'll still need to rebuild the image. No worries though, just something we need to know about. Thanks for filing the issue.